templates/install.tmpl

```diff
@@ -119,7 +119,7 @@
 					</div>
 					<div class="inline required field {{if .Err_RunUser}}error{{end}}">
 						<label for="run_user">{{.locale.Tr "install.run_user"}}</label>
-						<input id="run_user" name="run_user" value="{{.run_user}}" required>
+						<input id="run_user" name="run_user" value="{{.run_user}}" disabled>
```

Comment:
https://stackoverflow.com/questions/1355728/values-of-disabled-inputs-will-not-be-submitted

But, backend code still does `cfg.Section("").Key("RUN_USER").SetValue(form.RunUser)`

It will cause bugs.

Reply:
change disabled -> readonly.